Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -78,25 +80,32 @@ private void OnGasThermoRefreshParts(EntityUid uid, GasThermoMachineComponent co
                 switch (part.PartType)
                 {
                     case MachinePart.MatterBin:
+                        nBins += 1;
                         matterBinRating += part.Rating;
                         break;
                     case MachinePart.Laser:
+                        nLasers += 1;
                         laserRating += part.Rating;
                         break;
                 }
             }
+            laserRating /= nLasers;
+            matterBinRating /= nBins;
 
-            component.HeatCapacity = 5000 * MathF.Pow((matterBinRating - 1), 2);
+            component.HeatCapacity = 5000 * MathF.Pow(matterBinRating, 2);
 
             switch (component.Mode)
             {
-                // 573.15K with stock parts.
+                // 1172.6K with stock parts.
```

Comment:
this is a little high. 573.15k for stock was fine imo

Reply:
1172 was the actual value, the comment was just wrong. But yeah I can change it.

Reply:
for the record, the actual defaults before this were a min of 245K and a max of 2012K

Reply:
I think it used to be 0C +300k, but now its just  20C + 300k.